### PR TITLE
Edit soundboard 1

### PR DESCRIPTION
**Pull request by jkwok91**
Continuation of https://github.com/hackclub/hackclub/pull/817

TODOs:
- [x] need different sounds to inspire the use of keypress (for example, notes or sounds that lend more to layering (I picked a bad sample set and I am sorry))
- [ ] ~~add a way to kill audio if user decides to customize and use a very long mp3~~
- [x] add more inspiration / sample demo of this project

### CODE REVIEW

**Comment by zachlatta** [comment]
Any updates on the todos for this? Would love to merge as soon as possible.

**Comment by jkwok91** [comment]
Merge. Todos still relevant.

File: workshops/soundboard/README.md [hunk 1]
```diff
@@ -90,16 +89,63 @@ Now we have successfully hooked up the JavaScript to the HTML. Our `index.html`
...
+```js
+<body>
+  <div>
+
+  </div>
+
+  <script src="https://code.jquery.com/jquery-2.2.3.min.js"></script>
+  <script src="main.js"></script>
+</body>
+```
```
Comment (zachlatta) [inline workshops/soundboard/README.md]: Should be an HTML code block, not JavaScript.

File: workshops/soundboard/README.md [hunk 2]
```diff
@@ -90,16 +89,63 @@ Now we have successfully hooked up the JavaScript to the HTML. Our `index.html`
...
+The "class" attribute helps categorize similar elements. Since each of these will be rows in our soundboard, we'll give them each a class of "row":
+
+```js
+<div>
+  <div class="row">
+  </div>
+  <div class="row">
+  </div>
+</div>
+```
```
Comment (zachlatta) [inline workshops/soundboard/README.md]: Should be an HTML code block, not JavaScript.

File: workshops/soundboard/README.md [hunk 3]
```diff
@@ -90,16 +89,63 @@ Now we have successfully hooked up the JavaScript to the HTML. Our `index.html`
...
+Next, let's build the structure of our soundboard, by adding in rows to make a 2x8 grid. We'll do this by adding two `div` elements inside the `div` we just created:
+
+```js
+<div>
+  <div>
+  </div>
+  <div>
+  </div>
+</div>
+```
```
Comment (zachlatta) [inline workshops/soundboard/README.md]: Should be an HTML code block, not JavaScript.

**Comment by zachlatta** [comment]
:raised_hands:

**Pull request merged**